docker/Dockerfile

```diff
@@ -446,7 +446,7 @@ RUN apt-get -qq update && \
 # matplotlib improves plotting quality with better color maps and properly rendering colorbars.
 RUN dpkgArch="$(dpkg --print-architecture)"; \
     case "$dpkgArch" in amd64) \
-      pip3 install --no-cache-dir pyvista==${PYVISTA_VERSION} ;; \
+    pip3 install --no-cache-dir pyvista==${PYVISTA_VERSION} ;; \
```

Comment:
I thought we weren't installing pyvista in the container because it pulls in large dependencies?

Reply:
We install pyvista in the lab environment, But not pyvistaqt (which is the one that has large dependencies)